uint8 track_id
bool movable_fast
bool movable_slow
bool moving
int8 power
